bot/exts/filters/filter_lists.py

```diff
@@ -287,10 +293,58 @@ async def deny_sync(self, ctx: Context) -> None:
         """Syncs both allowlists and denylists with the API."""
         await self._sync_data(ctx)
 
+    @command(name="filter_report")
+    async def force_send_weekly_report(self, ctx: Context) -> None:
+        """Send a list of autobans added in the last 7 days to #mod-meta."""
```

Comment:
This docstring is technically inaccurate since it can send the report in any channel (you probably pasted this over from another method).
Also. we should probably add a check to ensure that the command cannot be run in public channels.

Reply:
Updated the docstring in [`f2d05d5` (#2267)](https://github.com/python-discord/bot/pull/2267/commits/f2d05d554f012598bcd42b4888400ced6513cb0d). There is a cog wide check which ensures commands are only ran in mod channel (which is why we can't modify filters elsewhere)

Reply:
The check only prevents non-mods from running the command as far as I can see, it doesn't impose any channel restriction.

Reply:
Good point, [`2142d82` (#2267)](https://github.com/python-discord/bot/pull/2267/commits/2142d824272c738c3d4ac9f20dd59cc4f02e345e)